book/meas_rules.qmd

```diff
@@ -10,296 +10,134 @@ abstract: TODO (150-200 WORDS)
+As well as being intuitive, when censoring is uninformative, the Graf score consistently estimates the mean square error $L(t, S|\tau^*) = \int^{\tau^*}_0 [\II(t > \tau) - S(\tau)]^2 d\tau$, where $S$ is the correctly specified survival function [@Gerds2006].
 However, despite these promising properties, the IGS is improper and must therefore be used with care [@Rindt2022; @Sonabend2022b].
-
-The reweighted IGS is a strictly proper outcome-independent loss [@Sonabend2022b] that reweights the IGS by removing censored observations and reweighting the denominator.
+In practice, experiments have shown that the effect of improperness is minimal and therefore this loss should be avoided for automated routines such as model tuning, however it can still be used for model evaluation.
+<!-- CITE OUR PROPERNESS PAPER HERE -->
+In addition, a small adaptation of the loss results in a strictly proper scoring rule simply by altering the weights such that $W_i = \KMG^{-1}(t_i)$ for all uncensored observations and $0$ otherwise [@Sonabend2022b], resulting in the reweighted Graf score:
+<!-- CITE OUR PROPERNESS PAPER HERE -->
 
 $$
-L_{RIGS}(\hatS, t, \delta|\KMG) = \frac{\delta \int_{\calT} (\II(t \leq \tau) - \hatF(\tau))^2 \ d\tau}{\KMG(t)}
+L_{RGS}(\hatS_i, t_i, \delta_i|\KMG) = \frac{\delta_i \int_{\calT} (\II(t_i \leq \tau) - \hatF_i(\tau))^2 \ d\tau}{\KMG(t_i)}
 $$
 
-This loss removes all censored observations, which can be problematic if the proportion of censoring is high.
-<!-- FIXME: THIS IS ALSO TERRIBLY WRITTEN! -->
-For uncensored observations we expect the predicted survival probability to be $1$ before any outcome is observed and $0$ otherwise, which follows more closely to the integrated Brier score.
-By changing the weighting the interpretation of contributions at time-points changes slightly, in the original IGS we may think of this as "inverse weighting for as long as the observation remains in the data", which means the weight of a contribution at a time-point will be different for all observations and all time-points.
-On the other hand, for RIGS, we weight by the outcome time for each observation, which remains the same over time.
-Hence we instead inflate scores for observations whose outcome are later in the dataset, this is intuitive as it essentially places more importance on observations that are representative of being alive at those time points.
+The change of weighting slightly alters the interpretation of the contributions at different time-points.
+By example, let $(t_\alpha = 4, t_\beta = 5)$ be two observed survival times, then at $\tau = 3$, the Graf score weighting would be $\KMG^{-1}(4)$ for both observations, whereas the RGS weights would be $(KMG^{-1}(4), KMG^{-1}(5))$ respectively, hence there is always more 'importance' placed on observations that take longer to experience the event.
```

Comment:
don't like the \alpha, \beta notation. these should be reserved for model parameters. maybe t and t' or \tilde{t}, or t_i, t_j